cmd/shell/terminal_test.go

```diff
@@ -0,0 +1,51 @@
+// +build !windows
```

Comment:
github.com/creack/pty has no windows support =\

Reply:
@danielsuguimoto OK! that is not worrying of this test... BUT, since we have `kool` being built and running on Windows, we should then add a `cmd/shell/terminal_windows.go` file with some variation for the implementation, without `creack/pty`... is there a different way we can tell this under Windows? Or maybe we can settle for having a environment variable only for Windows need.

Reply:
@fabriciojs actually, this package is used on terminal testing, only. It only mocks a pseudo TTY for testing purposes.

Reply:
AH sorry, that is correct. `moby/terminal` will be one handling TTY assertion under Windows, we are just not checking that ourselves. That sounds good!